### ISSUE TITLE
The keyboard user is unable to scroll horizontally the tree View elements (i.e., left pane of inspector window) using keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#292]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to Left pane (Tree View pane) of the inspector window.
3. Navigate through these tree items using arrow keys.
### Expected Behavior
The keyboard user should be able to scroll the entire left pane or tree view elements horizontally using keyboard.
### Actual Behavior
The keyboard user is unable to scroll the tree View elements (i.e., left pane of inspector window) horizontally using keyboard.
Note1: The User is able to scroll vertically these tree view elements using fn+up/down arrow keys or Control+ up/down arrow keys.
VoiceOver: The VoiceOver user is able to scroll horizontally Using VO keys (the Control and Option keys) + Arrow keys, after interacting (using Control, Option, shift and down arrow to interact) keys with the seprator.
### User Impact
The keyboard  user might face difficulty to see the lengthy names/labels of the tree view elements.

![kb8_leftpaneui-tree_nohorizontalscrolling](https://user-images.githubusercontent.com/22956130/39987317-053dc8ae-5782-11e8-95b7-47a4719ee6a1.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
